entity/src/sbom_package_license.rs

```diff
@@ -27,13 +28,31 @@ pub enum Relation {
     License,
 }
 
-#[derive(Copy, Clone, Debug, strum::Display, Eq, PartialEq, EnumIter, DeriveActiveEnum)]
+#[derive(Copy, Clone, Debug, strum::Display, Eq, PartialEq, EnumIter, Enum, DeriveActiveEnum)]
 #[sea_orm(rs_type = "i32", db_type = "Integer")]
 pub enum LicenseCategory {
     Declared = 0,
     Concluded = 1,
 }
 
+impl LicenseCategory {
+    pub fn to_i32(self) -> i32 {
+        self as i32
+    }
+}
+
+impl TryFrom<i32> for LicenseCategory {
+    type Error = &'static str;
```

Comment:
`&'static str` isn't a very good error type. With `thiserror` you can easily accomplish the same, but with a proper error type.